charts/kueue/templates/rbac/pending_workloads_CQ_viewer_role.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: '{{ include "kueue.fullname" . }}-pending-workloads-viewer-role'
+  name: '{{ include "kueue.fullname" . }}-pending-workloads-CQ-viewer-role'
```

Comment:
Do we use capital letters for other `ClusterRole` names? If not maybe use lower `lq` here as well, for consistency?